### ISSUE TITLE
Installer bundle minimal

### ISSUE THREAD

**Issue by akkermanronald** [akeneo/pim-community-standard#201]
After a clean composer install of 1.4.3 Akeneo on a clean Ubuntu 14.04 instance with installation parameter set at "PimInstallerBundle:minimal" we are unable to create families, attributes, products, etc. 

After pressing the "create ..." button the screen will hang forever and nothing will happen. When relaunching the same install with demo data with the php app/console pim:installer:db --env=prod command everything works fine and new families etc. can be created as expected.

Returning to minimal thereafter results in the same issue. Is there something in the minimal data creation which is not compatible with 1.4? 

Regards
Ronald

**Comment by jmleroux** [comment]
Thanks for your feedback Ronald. We will investigate on this issue with the minimal catalog.

Regards,
JM

**Comment by igormukhingmailcom** [comment]
Hi.

Have same issue.
As I see this is because `minimal` have `attributes.yml` and this file not imported (attributes table are empty after installation). But `icecat_demo` have `attributes.csv` and it working fine.

So I think that new akeneo don't use YAML format of attributes and attributes options.

**Comment by nidup** [comment]
You're right we tend to use only csv files in installer to use exact same format and validation than in imports, it has been fixed in last patch cf https://github.com/akeneo/pim-community-dev/tree/1.4/src/Pim/Bundle/InstallerBundle/Resources/fixtures/minimal

**Comment by igormukhingmailcom** [comment]
I'm using "today" `pim-community-standard` version with next deps at composer:

```
"require": {
        "akeneo/pim-community-dev": "v1.4.0-RC1",
        "sensio/generator-bundle": "2.3.5"
    },
    "require-dev": {
        "doctrine/migrations": "1.0.0-alpha3@alpha",
        "doctrine/doctrine-migrations-bundle": "1.0.0"
    },
```

Should I replace `v1.4.0-RC1` by `~1.4`?

**Comment by nidup** [comment]
Ok i got it! Master is the future 1.5 (we change it just before the release) you should use the 1.4 branch, that's why the composer.json is weird :) 

You can install with : php composer.phar create-project --prefer-dist akeneo/pim-community-standard ./pim-project "1.4.*@stable"

**Comment by buttash** [comment]
Also, the documentation says you should edit parameters.yml when you need to edit pim_parameters.yml...

**Comment by nidup** [comment]
Hi @akkermanronald 

To ensure a better follow-up of the issues, we're centralizing them in the community dev repository https://github.com/akeneo/pim-community-dev/issues.

If you didn't get the expected answer to this issue, please re-open it in the community dev repository and ping me (at nidup).

I will migrate the original discussion to avoid to loose information and context.

Sorry for the annoyance and thank you!
